cmds/webboot/utils.go

```diff
@@ -68,7 +68,7 @@ func download(URL, fPath string, uiEvents <-chan ui.Event) error {
 	}
 
 	counter.Close()
-	copyProgress := menu.NewProgress("Download complete. Writing ISO to cache", true)
+	copyProgress := menu.NewProgress("Download complete. Writing ISO to cache (" + fPath + ")", true)
```

Comment:
This is great, my suggestion is to look at using the `fmt.Sprintf` function with the `%q` qualifier.